appscale/tools/parse_args.py

```diff
@@ -375,6 +375,32 @@ def add_allowed_flags(self, function):
       self.parser.add_argument('--keyname', '-k',
         default=self.DEFAULT_KEYNAME,
         help="the keypair name to use")
+    elif function == "appscale-show-stats":
+      self.parser.add_argument('--keyname', '-k',
+        default=self.DEFAULT_KEYNAME,
+        help="the keypair name to use")
+      self.parser.add_argument('--show',
+        nargs='+',
+        choices=['nodes', 'processes', 'proxies'],
+        default=['nodes', 'proxies'],
+        help="the list of entities that statistics should be printed")
+      self.parser.add_argument('--roles', '-r',
+        nargs='*',
+        default=[],
+        help="print nodes with specified role(s)")
+      self.parser.add_argument('--order-processes', '-o',
+        nargs=1,
```

Comment:
Yes, this only allows one argument, but if I specify a choice list to the argument and string as a default type, it will return string in case when I  do not write this argument in a command line and return list in case when I do write this argument.
For example:
"... --show processes ..." returns 'cpu'
"... --show processes -o cpu ..." returns ['cpu']
So, I need to specify list as a default type.